code/game/objects/objs.dm

```diff
@@ -374,6 +374,12 @@
 /obj/proc/plunger_act(obj/item/plunger/P, mob/living/user, reinforced)
 	return
 
+/obj/proc/log_item(mob/user, actverb, notify_to_admin=FALSE, additional_info="")
+	if(notify_to_admin)
+		message_admins("[src] is [actverb] by [ADMIN_LOOKUPFLW(user)] at [ADMIN_VERBOSEJMP(user)]. [additional_info]")
```

Comment:
I dont think this should message admins directly at all, itw ill cause so much clutter